### DISCUSSION TITLE
Export excel [Jul 06]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2675] [Discord Threads]
> 🙋‍♂️ **Developer**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@964756055609458719> There is an option for download CSV. Only table currently working with will be downloaded as a CSV file.  
![image0](https://cdn.discordapp.com/attachments/993782464298373180/993805406734385232/Screenshot_2022-07-05_at_2.37.35_PM.png)  
#  
> 🙋‍♂️ **Developer**  
#### Yes I saw that but need to export in excel format  
#  
> 👨‍🦲 **dstala**  
#### Download in Excel format is not supported. Individual CSV filed downloaded per table needs to be imported onto excel (using spreadsheet/ MS excel) manually.  
#  
> 🙋‍♂️ **Developer**  
#### can we add that functionality in noco DB platform using some customization ?  
#  
> 🙋‍♂️ **wkw**  
#### please raise a feature request in github  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/993782464298373180)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
